modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -6,16 +6,18 @@ Issue Tracker Filter Form [Automation Testing]
 5. Test that the watching checkbox works correctly (issues that your userID is watching in issues_watching table)
 6. Check that links to issues in table are correct.
 7. Check that table sorts and displays additional pages correctly 
+8. Check that a user who does not have access to all centers can see all issues with a site set to NULL.
 
 Issue Tracker Create New Issue [Manual Testing]
 1. User can access the page if they have reporter permission
 2. Check that title and assignee are required. 
-3. Should display message, and redirect after success. 
-4. Submit invalid and valid PSCID and visit label pairs. Error messages should respond accordingly. Not that you cannot submit PSCIDs from other sites unless you have access all profiles permission
-5. Submit just a visit label - this should give an error message.
-6. Check that all values are propagated and saved correctly.
-7. Add an attachment to the new issue and make sure that it is successfully uploaded.
-8. Check that watching logging is working - turn it off and on for your current user, and for other watchers on the issue
+3. Check that site can be null or populated by one of the dropdown values. 
+4. Should display message, and redirect after success. 
+5. Submit invalid and valid PSCID and visit label pairs. Error messages should respond accordingly. Not that you cannot submit PSCIDs from other sites unless you have access all profiles permission
+6. Submit just a visit label - this should give an error message.
+7. Check that all values are propagated and saved correctly.
+8. Add an attachment to the new issue and make sure that it is successfully uploaded.
+9. Check that watching logging is working - turn it off and on for your current user, and for other watchers on the issue
```

Comment:
```suggestion
5. Submit invalid and valid PSCID and visit label pairs. Error messages should display accordingly. 
6. A user should be able to submit a PSCID from other sites only if they have access_all_profiles permission.  
7.  Submit a PSCID and leave the Site blank _ .. this should/not work if... _
8.  Submit just a visit label - this should give an error message.
9. Check that all values are propagated and saved correctly.
10. Add an attachment to the new issue and make sure that it is successfully uploaded.
11. Check that watching logging is working - turn it off and on for your current user, and for other watchers on the issue.
```

Reply:
Breaking down testing point 5.  
- [ ] To do : Point 7 - needs to be completed with what should be happening when a PSCID is entered with Null site.

Reply:
@christinerogers Are 7. and 3. redundant?

Reply:
I see your point.  
3 could specify that PSCID is left blank.  
 7 tests the case where there is a PSCID, and Site doesn't match or is null.